backstop/tests/tree.html

```diff
@@ -0,0 +1,1305 @@
+            type="button"
+          >
+            <svg-chevron-right
+              class="iui-icon"
+              aria-hidden="true"
+            ></svg-chevron-right>
+          </button>
+          <span class="iui-node-label">
+            <div class="iui-title">Subcategory</div>
+          </span>
+        </div>
+        <div
+          class="iui-tree-node"
+          style='--level: 1;'
+          tabindex="0"
+        >
+          <button
+            class="iui-button iui-borderless iui-small"
+            type="button"
+          >
+            <svg-chevron-right
+              class="iui-icon"
+              aria-hidden="true"
+            ></svg-chevron-right>
+          </button>
+          <span class="iui-node-label">
+            <div class="iui-title">Subcategory</div>
+          </span>
+        </div>
+        <div
+          class="iui-tree-node"
+          style='--level: 1;'
+          tabindex="0"
+        >
+          <span class="iui-node-label">
+            <div class="iui-title">Subcategory</div>
+          </span>
+        </div>
+        <div
+          class="iui-tree-node iui-expanded"
+          tabindex="0"
+        >
+          <button
+            class="iui-button iui-borderless iui-small"
+            type="button"
+          >
+            <svg-chevron-right
+              class="iui-icon"
+              aria-hidden="true"
+            ></svg-chevron-right>
+          </button>
+          <span class="iui-node-label">
+            <div class="iui-title">Category 2</div>
+            <div class="iui-caption">A basic description.</div>
+          </span>
+        </div>
+        <div
+          class="iui-tree-node"
+          style='--level: 1;'
+          tabindex="0"
+        >
+          <button
+            class="iui-button iui-borderless iui-small"
+            type="button"
+          >
+            <svg-chevron-right
+              class="iui-icon"
+              aria-hidden="true"
+            ></svg-chevron-right>
+          </button>
+          <span class="iui-node-label">
+            <div class="iui-title">Subcategory</div>
+          </span>
+        </div>
+        <div
+          class="iui-tree-node"
+          tabindex="0"
+        >
+          <button
+            class="iui-button iui-borderless iui-small"
+            type="button"
+          >
+            <svg-chevron-right
+              class="iui-icon"
+              aria-hidden="true"
+            ></svg-chevron-right>
+          </button>
+          <span class="iui-node-label">
+            <div class="iui-title">Category 3</div>
+            <div class="iui-caption">A basic description.</div>
+          </span>
+        </div>
+      </div>
+    </section>
+
+    <hr />
+    <h2>With Checkbox</h2>
+    <section id="demo-checkbox">
+      <div class="iui-tree">
+        <div
+          class="iui-tree-node iui-expanded"
+          tabindex="0"
+        >
+          <label class="iui-checkbox iui-checkbox-visibility">
```

Comment:
Why eye background is not transparent?

![image](https://user-images.githubusercontent.com/36186912/142984929-b4726eb8-20b2-4a81-bcbf-44462c7cd1ba.png)

Reply:
Fixed in #392.